aea/cli/add.py

```diff
@@ -141,7 +141,9 @@ def _add_item(click_context, item_type, item_public_id) -> None:
         )
     else:
         package_path = fetch_package(item_type, public_id=item_public_id, cwd=ctx.cwd)
+
     if item_type in {"connection", "skill"}:
+        # TODO: find a better way to handle dependencies
```

Comment:
That is a pretty pointless TODO..it should be specific or removed. General requests to refactor don't warrant a TODO.